### PR TITLE
Add yuagao shanghai recommendations

### LINKED ISSUE #18: Add yuagao Shanghai file
- [x] Create a **branch**
- [x] Add file/commits
- [x] Open a _PR_
- [x] Have a discussion about the PR(COLLABORATE)
- [x] Make edits
- [ ] Get :shipit: approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by yuagao**
closes #18

@beardofedu please check my code 👋 

thanks 😄

### CODE REVIEW

**Comment by beardofedu** [comment]
@yuagao Are there any brewery tours that I can go on in Shanghai?

**Comment by yuagao** [comment]
@beardofedu sry man, I don't know 😦 I know some good brewery tours in Beijing, and you can visit later 🍻 🍻

File: yuagao-shanghai.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Shanghai, China
+
+###Places to Eat
```
Comment (yuagao) [inline yuagao-shanghai.md:3]: good

**Comment by beardofedu** [comment]
Sounds like a plan @yuagao !!! Thanks for the info!

**Pull request merged**